Source/app/routes.js

```diff
@@ -72,6 +72,26 @@ module.exports = function(app, passport) {
         })
     );
 
+    //=====================================
+    // Data Collection
+    //=====================================
+    // Collects POST data from acquisuites,
+    // converts it to JSON, and saves it to
+    // the database.
+    app.post('/acquisuite/upload/:id', function (req, res) {
```

Comment:
will req be coming from web socket server/db server?

Reply:
As far as I know, the request will come from the web server. I'm using Express's built in req object ([more on that HERE](http://expressjs.com/en/api.html#req)).

Reply:
We used this method to collect POST data in CS 290, anyway. It was pretty straightforward.

Reply:
Right. But the $HTTP.posts will probably becoming from a web socket server or DB server that we set up, isn't that what we agreed upon?

Reply:
We use the same method to receive posts from the Angular services as well. The implementation is correct, just curious if you had a plan for where that post was coming from.

Reply:
Are you available to hop on Hangouts for a sec?

Reply:
It's no big deal but I might have misunderstood how we were structuring the data collection.